odoo/tools/safe_eval.py

```diff
@@ -238,131 +825,134 @@ def const_eval(expr):
     >>> const_eval("1+2")
     Traceback (most recent call last):
     ...
-    ValueError: opcode BINARY_ADD not allowed
     """
-    c = test_expr(expr, _CONST_OPCODES)
-    return unsafe_eval(c)
+    return ast.literal_eval(expr)
+
 
 def expr_eval(expr):
```

Comment:
this is used at only one place, is it possible to remove that function ? (and use a simple safe_eval ?)
(to check if it's safe to do)

Reply:
I don't know, maybe it's used in some custommer's custom code :thinking:

Reply:
dont be afraid to break thing if it make community better (you will annoy people anyway with all thoses changes)